README.md

```diff
@@ -19,140 +19,251 @@ it, other platforms do not require it.
+      - name: do-ip4
+        value: "yes"
+      - name: do-ip6
+        value: "no"
+      - name: verbosity
+        value: 0
+      - name: username
+        value: "{{ nsd_user }}"
+      - name: zonesdir
+        value: '"{{ nsd_conf_dir }}"'
+      - name: database
+        value: '"{{ nsd_db_dir }}/nsd.db"'
+      - name: pidfile
+        value: '"{{ nsd_run_dir }}/nsd.pid"'
+      - name: xfrdfile
+        value: '"{{ nsd_state_dir }}/xfrd.state"'
+      - name: hide-version
+        value: "yes"
+```
+
+## `nsd_config_remote_control`
+
+This variable is a list of dict or string for `remote-control` directive in
+`nsd.conf(5)`.
+
+The same rules describe in `nsd_config_server` apply.
+
+## `nsd_zones`
+
+This variable is a dict of zones. Keys are domain name of zones. Each key must
+have a dict as a value, and explained below.
+
+| Key | Value | Mandatory? |
+|-----|-------|------------|
+| `zonefile` | relative path to the zone file from `nsd_conf_dir` | no |
+| `zone` | the zone definition | no |
+| `config` | list of configurations of the zone (see below) | no |
+
+`config` is a list of configurations for the zone. An element can be a dict or
+a string.
+
+When an element of `config` is a dict, the dict must have a mandatory key,
+`name`, which is one of zone options described in `nsd.conf(5)`. The dict must
+have either `values` or `value` as a key. `values` is a list of string values
+for the option. Use `values` for options that are allowed to appear multiple
+times.  `value` is a string for the option.
+
+When an element of `config` is a string, the string is simply added to the zone
+configuration.
+
+An example:
+
+```yaml
+nsd_zones:
+  example.com:
+    zonefile: example.com.zone
+    zone: |
+      example.com. 86400 IN SOA ns1.example.com. hostmaster.example.com. 2013020201 10800 3600 604800 3600
+      example.com. 86400 IN NS ns1.example.com.
+      example.com. 120 IN A 192.168.0.1
+      ns1.example.com. 120 IN A 192.168.0.1
+      mail.example.com. 120 IN A 192.168.0.1
+      example.com. 120 IN MX 25 mail.example.com.
+    config:
+      - name: provide-xfr
+        values:
+          - 192.168.133.101 my_tsig_key
+          - 127.0.0.1 NOKEY
+      - name: allow-notify
+        value: 192.168.0.111 NOKEY
+      - name: request-xfr
+        values:
+          - 192.168.0.111 NOKEY
+      - name: outgoing-interface
+        values:
+          - 192.168.0.1
+      - 'allow-axfr-fallback: yes'
+```
 
 ## Debian
 
 | Variable | Default |
 |----------|---------|
-| \_\_nsd\_user | nsd |
-| \_\_nsd\_group | nsd |
-| \_\_nsd\_db\_dir | /var/lib/nsd |
-| \_\_nsd\_conf\_dir | /etc/nsd |
-| \_\_nsd\_run\_dir | /var/run/nsd |
-| \_\_nsd\_state\_dir | /var/lib/nsd |
+| `__nsd_user` | `nsd` |
+| `__nsd_group` | `nsd` |
+| `__nsd_db_dir` | `/var/lib/nsd` |
+| `__nsd_conf_dir` | `/etc/nsd` |
+| `__nsd_run_dir` | `/var/run/nsd` |
+| `__nsd_state_dir` | `{{ __nsd_db_dir }}` |
```

Comment:
`s/__nsd_db_dir/nsd_db_dir/` or users must modify `nsd_state_dir` if `nsd_db_dir` is set.